src/pages/components/productlist.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { useState } from 'react';
+import { useEffect } from 'react';
+import Product from './Product';
+import './productlist.scss';
+import './product.scss';
+import { Link } from 'react-router-dom';
```

Comment:
```suggestion
import React, {useState, useEffect} from 'react';
import { Link } from 'react-router-dom';
import Product from './Product';
import './productlist.scss';
import './product.scss';
```

이렇게 임포트 순서에 맞춰서도 사용할 수 있겠습니다!